examples/vision/semisupervised_simclr.py

```diff
@@ -199,24 +200,24 @@ def get_config(self):
 
     def call(self, images, training=True):
         if training:
-            batch_size = tf.shape(images)[0]
+            batch_size = ops.shape(images)[0]
 
             # Same for all colors
-            brightness_scales = 1 + tf.random.uniform(
+            brightness_scales = 1 + keras.random.uniform(
                 (batch_size, 1, 1, 1),
                 minval=-self.brightness,
                 maxval=self.brightness,
             )
             # Different for all colors
-            jitter_matrices = tf.random.uniform(
+            jitter_matrices = keras.random.uniform(
                 (batch_size, 1, 3, 3), minval=-self.jitter, maxval=self.jitter
             )
 
             color_transforms = (
                 tf.eye(3, batch_shape=[batch_size, 1]) * brightness_scales
```

Comment:
Can this be converted to `ops`?

Reply:
Yeah, actually, there were some issues. In spite of having the same TF ops in Keras, the ```batch_shape``` attribute that is present here is missing in Keras ops. That is why I did not change this in the first place. But I addressed this in the latest commit and replaced the TF ops.